app/templates/macros/file_upload.html

```diff
@@ -0,0 +1,95 @@
+{% macro file_upload(fieldname) %}
+<script>
+$('body').on('change', 'input:file', function() {
+  var file = $(this)[0].files[0];
+  getSignedRequest(file)
+})
+function getSignedRequest(file){
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', `/sign-s3?file-name=${file.name}&file-type=${file.type}`);
+  xhr.onreadystatechange = function() {
+  if (xhr.readyState === 4) {
+    if (xhr.status === 200){
+      var response = JSON.parse(xhr.responseText);
+      console.log("response form json dumps: ", response);
+      uploadFile(file, response.data, response.url, response.url_upload)
+    } else{
+    alert('Could not get signed URL.');
+    }
+  }
+};
+  xhr.send();
+}
+function uploadFile(file, s3Data, url, urlUpload, fieldName) {
```

Comment:
I get this error when I try to upload a photo!
<img width="626" alt="screen shot 2018-02-24 at 11 32 08 am" src="https://user-images.githubusercontent.com/9835059/36632604-64c33c24-1956-11e8-8c71-61540e1b410c.png">

Reply:
You need to have the config.env for it to work. I'll give you credentials at the meeting :)